### ISSUE TITLE
App keep loading no error show (Bash on Windows)

### ISSUE THREAD

**Issue by koalabear974** [sj26/mailcatcher#328]
Hi,
So I tried to setup my ruby environment on this new bash subsystem on Windows.
My app work right and all the gem was compatible.
Comes to mailcatcher installed from gem on this bash subsystem, no error shown on the installation.
No error shown also on the launch :
`Starting MailCatcher
==> smtp://127.0.0.1:1025
==> http://127.0.0.1:1080`
but when I'm trying to load the page, it never stop loading until I quit the process by hand in the bash console.
So basically no error shown on console and on chrome console, the page never show but keep loading. (Sometimes when I stop the daemon the page stop loading and display some HTML code it apparently retrieve)

It would be nice to have mailcatcher working on the bash subsystem since it's the environment advised by many sources now. You might want to see this page to reproduce the bug maybe https://gorails.com/setup/windows/10

Thanks for your help,

**Comment by sj26** [comment]
Hi @koalabear974, I don't have a Windows system to get this running myself but if you want to contribute a patch it would be most welcome! Otherwise I'm afraid I'm unable to fix this and will close.
